src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -22,19 +29,70 @@
    * the database and checking to make sure its valid and not null
    * and sending this information to the webpage
    */
+   
+  // Store class that gives access to Users. 
+  private UserStore userStore;
+  
+  // Store class that gives access to Conversations.
+  private ConversationStore conversationStore;
+  
+  // Store class that gives acces to messages
+  private MessageStore messageStore;
   
   /** This is for the inital setup of the admin page */
   @Override
   public void init() throws ServletException {
     super.init();
+    
+    setUserStore(UserStore.getInstance());
+    setConversationStore(ConversationStore.getInstance());
+    setMessageStore(MessageStore.getInstance());
+  }
+  
+  /**
+   * Sets the UserStore used by this servlet. 
+   * This function provides a common setup method for use
+   * by the test framework or the servlet's init() function.
+   */
+  void setUserStore(UserStore userStore) {
+    this.userStore = userStore;
+  }
+  
+  /**
+   * Sets the ConversationStore used by this servlet. 
+   * This function provides a common setup method
+   * for use by the test framework or the servlet's init() function.
+   */
+  void setConversationStore(ConversationStore conversationStore) {
+    this.conversationStore = conversationStore;
+  }
+  
+  /**
+   * Sets the MessageStore used by this servlet. 
+   * This function provides a common setup method
+   * for use by the test framework or the servlet's init() function.
+   */
+  void setMessageStore(MessageStore messageStore) {
+    this.messageStore = messageStore;
   }
   
   /** when user visits this sends them to admin page */
   @Override
   public void doGet(HttpServletRequest request, 
       HttpServletResponse response)
       throws IOException, ServletException {
-    String requestUrl = request.getRequestURI();
+    
+    /** This gets the current number of users in the database */
```

Comment:
Please see the comment I made before about comments. Single line comments that are not docstrings can use // and you can delete these as they are unnecessary